packages/kit/src/core/config/index.spec.js

```diff
@@ -87,6 +87,15 @@ test('errors on invalid nested values', () => {
 	}, /^Unexpected option config\.kit\.files\.potato$/);
 });
 
+test('does not error on invalid top-level values', () => {
+	assert.not.throws(() => {
+		validate_config({
+			// @ts-expect-error
```

Comment:
it might be nice to add a comment here as to why we expect an error

Reply:
@ignatiusmb helped adding that. thanks!